### ISSUE TITLE
Keyboard focus lost in git changes list when file is renamed

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#6635]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.3.938 
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
- Open a project that is under source control (git)
- Make various edits
- Rename an existing file
- Click on Commit button to bring up VCS window with list of modified/new/deleted files
- Click in that list to give it focus
- Use up/down arrows and spacebar to check/uncheck items
- Select the renamed file, and the deleted file representing "before" state
- Git recognizes this as a rename and collapses it to one item

### Describe the problem in detail
- Keyboard focus goes back to the start of the list

### Describe the behavior you expected
- Focus should stay on the single item that now represents the rename

Closely related bug: have to click in the list to give it focus, can't currently Tab to that list so a keyboard-only user is stuck and cannot use the feature.
